simplivity/resources/policies.py

```diff
@@ -73,6 +73,23 @@ def get_by_data(self, data):
         """
         return Policy(self._connection, self._client, data)
 
+    def create(self, name, flags=None, timeout=-1):
+        """ create the new policy
+
+        Args:
+            flags: Dictionary of filters, example: {'cluster_group_id': 'cluster_group_id'}
+            name : name of policy
+            timeout : timeout for the operation to complete
+
+        Returns:
+            Returns Policies object, list of policy which will always have single
+            policy created
+        """
+        data = {"name": name}
+        self._client.do_post(URL, data, timeout, None, flags)
```

Comment:
do_post returns the affected object id(id of the Policy created), let's use that to create an object of the Policy and return it. Create should return a policy object(creates only one Policy), not list.

       affected_object = self._client.do_post(URL, data, timeout, None)[0]
        return self.get_by_id(affected_object["object_id"])

Reply:
Thanks for code snippet. We tried these options yesterday, and for some reason it was not working. 
We tried everything using get_by_id as well as get_by_name. We were able to get the specific object in code however we were not able to return the object to example.py. 

However now we tried it and it is working fine.  Please find the updated code here. 
Please review and comment.